### PR TITLE
top 5 unexplained terms command

### PR DESCRIPTION
**Pull request by muhammadbassiony**
New command: '/explain top-5' to fetch the top 5 unexplained terms that were searched for.
Any new unexplained term is saved and a counter is initialized for it.

### CODE REVIEW

File: scripts/explain.js [hunk 1]
```diff
@@ -101,6 +104,19 @@ async function fetchExplanation(explainQuery) {
...
+  } else if (explainQuery.toLowerCase().trim() == "top-5") {
+    const unexplained_top = await unexplainedTop(doc);
+
+    let unexplained_str = "Top 5 unexplained terms. Format: '<term>(count)' ";
+    unexplained_top.forEach((u) => {
+      unexplained_str = unexplained_str + ` - \'${u.Explain}(${u.Count})\'`;
+    });
+
+    unexplained_str =
+      unexplained_str +
```
Comment (andesol) [inline scripts/explain.js:116]: += ? (here and above)

File: scripts/explain.js [hunk 2]
```diff
@@ -101,6 +104,19 @@ async function fetchExplanation(explainQuery) {
     const whyRandomRow = rows.find((row) => row.rowIndex == randomRowIndex);
     const display_text = whyRandomRow.why;
     return display_text;
+  } else if (explainQuery.toLowerCase().trim() == "top-5") {
+    const unexplained_top = await unexplainedTop(doc);
```
Comment (andesol) [inline scripts/explain.js:108]: camelCase?

**Comment by amylily1011** [comment]
LGTM :shipit:

**Comment by gerwitz** [comment]
I just tried this hoping it'd work, and then found this is still lingering. :shipit:

**Pull request merged**